tests/training/trainer_test.py

```diff
@@ -89,3 +109,113 @@ def forward(self, **kwargs):  # pylint: disable=arguments-differ,unused-argument
                               self.iterator, self.instances,
                               num_epochs=2, serialization_dir=self.TEST_DIR)
             trainer.train()
+
+    @pytest.mark.skipif(os.uname().sysname == 'Darwin', reason="Tensorboard logging broken on mac.")
```

Comment:
We need to find a solution to this - it's too big a thing to leave in a test. What went wrong?

Reply:
It's too hard to fix / not worth the time.  It's easier to remove the test.  This test runs on Team City.  The tensorboard dependency we are using has some issues.  The code installed by the pip version is different then the Linux version and the bug isn't present in the Linux version. And building it from scratch / installing from source seems overly complicated for what we get. We may eventually want to replace it with something else.

Reply:
It looks like the tensorboard dependency isn't being maintained on pip anymore:
https://github.com/dmlc/tensorboard/issues/50

But installing from the master branch requires bazel and what appears to be a multi-step build process:
https://github.com/dmlc/tensorboard

Reply:
Yeah ok that looks really nasty - can you open an issue in allennlp that this needs to be addressed though, because we need to update to whatever the new pypi version of that guy's repo is once it get's decided.

Reply:
done